src/main/java/com/ibm/nfvodriver/config/KafkaConfiguration.java

```diff
@@ -17,32 +19,32 @@
 @Configuration("KafkaConfiguration")
 public class KafkaConfiguration {
 
-	private static final Logger logger = LoggerFactory.getLogger(KafkaConfiguration.class);
-
-	@Configuration("KafkaConfigurationEnabled")
-	@ConditionalOnProperty(value = "vnfmdriver.kafka.enabled", matchIfMissing = true)
-	@EnableKafka
-	public static class KafkaConfigurationEnabled {
-
-		@Bean
-		@Primary
-		public ExternalMessagingService getKafkaEMS(NFVODriverProperties properties, KafkaTemplate<String, String> kafkaTemplate, ObjectMapper objectMapper) {
-			logger.info("Creating Kafka EMS Bus Connector");
-			return new KafkaExternalMessagingServiceImpl(properties, kafkaTemplate, objectMapper);
-		}
-
-//        @Bean
-//        public LcmOpOccPollingService lcmOpOccPollingService(NSLifecycleManagementDriver driver, ExternalMessagingService externalMessagingService, ObjectMapper objectMapper) {
-//            return new LcmOpOccPollingService(driver, externalMessagingService, objectMapper);
-//        }
-	}
-
-	@Bean
-	@ConditionalOnMissingBean
-	public ExternalMessagingService getLoggingEMS() {
-		logger.info("Creating Logging EMS Bus Connector");
-		return new LoggingExternalMessagingServiceImpl();
-	}
+    private static final Logger logger = LoggerFactory.getLogger(KafkaConfiguration.class);
+
+    @Configuration("KafkaConfigurationEnabled")
+    @ConditionalOnProperty(value = "vnfmdriver.kafka.enabled", matchIfMissing = true)
```

Comment:
This is still vnfmdriver instead of nfvodriver. Please do check where this configuration is used in the repo and correct it accordingly.

Reply:
changed the code and committed the code